### PR TITLE
Documentation for StackAdapt destination

### PR DESCRIPTION
**Pull request by AlliterativeAlice**
### Proposed changes

This PR adds documentation for the StackAdapt destination merged in https://github.com/segmentio/action-destinations/pull/1828

### Merge timing
As long as our destination will remain hidden from the catalog listing until our destination completes private beta and reaches general availability, the docs can be merged ASAP after approval.

### Related issues (optional)

https://github.com/segmentio/action-destinations/pull/1828 (PR that added destination this documentation is for)

### CODE REVIEW

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+3. In the instructions that appear, copy the universal pixel ID from the code snippet. Below is an example of a code snippet where the universal pixel ID is `sqQHa3Ob1hFi__2EcYYVZg1`.
+
+![Image showing location of universal pixel ID in code snippet](images/copy-pixel-id.png)
+
+### Setting up the StackAdapt destination in Segment
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Search for "StackAdapt" in the search bar, then click on the Destination "StackAdapt".
+3. Click **Add Destination**.
+4. Select an existing JavaScript Source to connect to TikTok Pixel.
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:34]: hi @AlliterativeAlice we should remove the reference to TikTok Pixel ;)

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]
Hi @AlliterativeAlice thanks for putting together this doc. The Segment Docs team will do a proper review. I noticed a reference to TikTok which we should change.

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+By setting up StackAdapt as a Segment destination, your Segment events will be forwarded to [StackAdapt](https://www.stackadapt.com/){:target="_blank"}. This allows you to generate retargeting and lookalike audiences, track conversions, and measure return on ad spend using your Segment events - bypassing the need to install the StackAdapt pixel on your website and write code to send events to StackAdapt.
+
+This destination is maintained by StackAdapt. For any issues with the destination, please [submit a ticket to StackAdapt's support team](https://support.stackadapt.com/hc/en-us/requests/new?ticket_form_id=360006572593){:target="_blank"}.
+
+{% include content/ajs-upgrade.md %}
+
+## Getting started
+
+### Getting Your StackAdapt Universal Pixel ID
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:18]:
```suggestion
### Getting your StackAdapt Universal Pixel ID
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+1. Log in to your StackAdapt account and navigate to the Pixels page.
+2. Above the list of pixels, click the "Install StackAdapt Pixel" link.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:21]:
```suggestion
2. Above the list of pixels, click **Install StackAdapt Pixel**.
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    ![Image showing location of link to install Pixel](images/install-pixel-link.png)
+
+3. In the instructions that appear, copy the universal pixel ID from the code snippet. Below is an example of a code snippet where the universal pixel ID is `sqQHa3Ob1hFi__2EcYYVZg1`.
+
+![Image showing location of universal pixel ID in code snippet](images/copy-pixel-id.png)
+
+### Setting up the StackAdapt destination in Segment
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Search for "StackAdapt" in the search bar, then click on the Destination "StackAdapt".
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:32]:
```suggestion
2. Search for and select the "StackAdapt" destination.
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+    ![Image showing location of link to install Pixel](images/install-pixel-link.png)
+
+3. In the instructions that appear, copy the universal pixel ID from the code snippet. Below is an example of a code snippet where the universal pixel ID is `sqQHa3Ob1hFi__2EcYYVZg1`.
+
+![Image showing location of universal pixel ID in code snippet](images/copy-pixel-id.png)
+
+### Setting up the StackAdapt destination in Segment
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:31]:
```suggestion
1. From the Segment web app, navigate to **Connections > Catalog > Destinations**.
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+2. Search for "StackAdapt" in the search bar, then click on the Destination "StackAdapt".
+3. Click **Add Destination**.
+4. Select an existing Source to connect to the StackAdapt destination.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:34]:
```suggestion
4. Select an existing source to connect to the StackAdapt destination.
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+5. Give the Destination a name.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:35]:
```suggestion
5. Give the destination a name.
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+6. On the Settings screen, provide your StackAdapt Universal Pixel ID. This can be found on the Pixels page in StackAdapt as described above.
+7. Toggle on the Destination using the **Enable Destination** toggle.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:37]:
```suggestion
7. Toggle on the destination using the **Enable Destination** toggle.
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+8. Click **Save Change**.
+
+### StackAdapt Pixel Setup
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:40]:
```suggestion
### StackAdapt Pixel setup
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+- [Creating Retargeting Audiences](https://support.stackadapt.com/hc/en-us/articles/360005939153-Creating-Retargeting-Audiences){:target="_blank"}
+- [How to Generate and Target a Lookalike Audience](https://support.stackadapt.com/hc/en-us/articles/360023738733-How-to-Generate-and-Target-a-Lookalike-Audience){:target="_blank"}
+
+Setup of conversion events, retargeting audiences, and lookalike audiences that fire on Segment events is largely the same as the setup in the StackAdapt documentation, with a few caveats:
+
+1. You **must** select "Universal Pixel" as the pixel type. This is because the StackAdapt destination in Segment uses your Universal Pixel ID to send events to StackAdapt.
+2. There is no need to install the StackAdapt pixel on your website as instructed in the "Installation" step, since Segment will forward events to StackAdapt that would normally be tracked by the StackAdapt pixel.
+3. If you choose to set up event rules, you will need to ensure that you use the event keys supported by the the StackAdapt destination as described below.
+
+### Event Rules
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:54]:
```suggestion
### Event rules
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+
+The StackAdapt Segment destination sends an `action` event key which by default is mapped to the Segment event name. Creating rules on this `action` key should be sufficient for most simple event rule use cases. For example, if you fire a Segment event when a user fills out a registration form on your website and want to track this as a conversion event in StackAdapt, you can create a rule in StackAdapt that matches the `action` key with the Segment event name.
+
+A Segment event fired with the code `analytics.track("User Registered")` can be tracked as a conversion event with an event rule that matches an `action` of `User Registered` as shown below:
+
+![Image showing event rule in StackAdapt the matches a User Registered event](images/user-registered-event-rule.png)
+
+#### Ecommerce Events
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:62]:
```suggestion
#### Ecommerce events
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+|----------------------------------|----------------------|
+| `order_id`                       | `order_id`           |
+| `revenue`                        | `revenue`            |
+| `product_id`                     | `product_id`         |
+| `category`                       | `product_category`   |
+| `name`                           | `product_name`       |
+| `price`                          | `product_price`      |
+| `quantity`                       | `product_quantity`   |
+
+For events that can involve multiple products, such as checkout events, we forward a JSON array of product objects with a `products` key and fields that map by default to following Segment product array fields:
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:76]:
```suggestion
For events that can involve multiple products, such as checkout events, StackAdapt forwards a JSON array of product objects with a `products` key and fields that map by default to following Segment product array fields:
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+      name: 'Monopoly: 3rd Edition',
+      price: 11.5,
+      quantity: 1,
+      category: 'Games'
+    }
+  ]
+});
+```
+
+### URL Rules
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:108]:
```suggestion
### URL rules
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+
+If you are using URL rules, these will be matched whenever Segment sends an event to StackAdapt with a `url` matching the URL rule. This should be accomplished by the page event Segment automatically fires when a page is viewed, so setup of URL rules should be identical to setting up URL rules with the StackAdapt pixel.
+
+{% include components/actions-fields.html %}
+
+## Data and Privacy
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:114]:
```suggestion
## Data and privacy
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
Thanks for submitting this, @AlliterativeAlice! I left a few minor edits to align this a bit more with our style guide.

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
+---
+title: StackAdapt (Actions) Destination
+hide-boilerplate: true
+hide-dossier: true
+beta: true
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:6]:
```suggestion
---
title: StackAdapt (Actions) Destination
hide-boilerplate: true
hide-dossier: true
beta: true
id: 61d8859be4f795335d5c677c
hidden: true
---
```

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+
+{% include content/plan-grid.md name="actions" %}
+
+By setting up StackAdapt as a Segment destination, your Segment events will be forwarded to [StackAdapt](https://www.stackadapt.com/){:target="_blank"}. This allows you to generate retargeting and lookalike audiences, track conversions, and measure return on ad spend using your Segment events - bypassing the need to install the StackAdapt pixel on your website and write code to send events to StackAdapt.
+
+This destination is maintained by StackAdapt. For any issues with the destination, please [submit a ticket to StackAdapt's support team](https://support.stackadapt.com/hc/en-us/requests/new?ticket_form_id=360006572593){:target="_blank"}.
+
+{% include content/ajs-upgrade.md %}
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:14]:
```suggestion
```

This is no longer needed

**Review by tcgilbert — CHANGES_REQUESTED** [review]
Just a couple of changes for me

**Review by rchinn-segment — APPROVED** [review]
lgtm! I'll leave this open for any final feedback, and be sure to include it in our next deploy (2/13)

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**